src/style.css

```diff
@@ -0,0 +1,24 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.star {
+  display: flex;
+}
+
+.stars__star {
+  width: 15px;
+  height: 15px;
+  margin-right: 4px;
+  background-image: url("./images/star.svg");
+  background-repeat: no-repeat;
```

Comment:
Try adding `background-position: center;` here.

Reply:
Done! Thank you:)